cloud/core/README.md

```diff
@@ -0,0 +1,80 @@
+# Cloud Core API
```

Comment:
Excellent!

Can you make line 1 look like:

https://github.com/artificialwisdomai/origin/pull/82/files#diff-288302a25f50e8bff08cdebce0d4e9253f36d379e8a71326dfacb29f491fa5f6R1